src/width.rs

```diff
@@ -1082,6 +1086,38 @@ pub(crate) fn wrap_text(text: &str, width: usize, keep_words: bool) -> String {
     }
 }
 
+#[cfg(feature = "color")]
+pub(crate) fn wrap_text(text: &str, width: usize, keep_words: bool) -> String {
+    if width == 0 {
+        String::new()
+    } else {
+        let (text, url): (String, Option<String>) = strip_osc(text);
+        let (prefix, suffix) = if let Some(url) = url {
+            // https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
+            let osc8 = "\x1b]8;;";
+            let st = "\x1b\\";
+            (format!("{osc8}{url}{st}"), format!("{osc8}{st}"))
```

Comment:
I can remove the unnecessary allocation here.

Reply:
I can imagine how it will look like :sweat_smile:

I guess we can leave it as it is (till better days) to not make *split_* function more complex.